frame/uniques/src/lib.rs

```diff
@@ -908,7 +908,7 @@ pub mod pallet {
 
 		/// Approve an item to be transferred by a delegated third-party account.
 		///
-		/// Origin must be Signed and must be the owner of the `item`.
+		/// Origin must be Signed and must be the owner of the `item` or the admin.
```

Comment:
```suggestion
		/// The origin must conform to `ForceOrigin` or must be `Signed` and the sender must be the
		/// owner of the `item`.
```

Reply:
Actually, this is not what I meant, in the function, there is a check whether the origin is the owner of the item or if the origin is the admin of the collection.

Reply:
Changed the wording. We should also include `ForceOrigin`. As you see this is also done in the other docs here.